configuration/init/systemd/tedge-agent.service

```
@@ -8,6 +8,7 @@ RuntimeDirectory=tedge_agent
 ExecStart=/usr/bin/tedge_agent
 Restart=on-failure
 RestartPreventExitStatus=255
+WatchdogSec=15
```

```suggestion
WatchdogSec=5
```
Or may be even 10 would be fine.